### PR TITLE
KTL-583: kikoff project with Gatsby

### PR DESCRIPTION
**Pull request by zoobestik**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
+  "name": "kotlinfoundation.github.io",
+  "version": "1.0.0",
+  "description": "Kotlin Foundation – official site",
+  "author": "zoobestik <kb.chernenko@gmail.com>",
```
Comment (nikpachoo) [inline package.json:5]: I guess it should be removed. 
```suggestion
```

File: src/components/Layout/index.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+                {children}
+            </article>
+
+            {!withoutCta && (
+                <CtaBlock
+                    topTitle={<>
+                        Have any questions? <br/>Contact us!
+                    </>}
+                    mainTitle={
+                        <a href="mailto:hello@kotlinfoundation.org" className="layout__cta-link">hello@kotlinfoundation.org</a>
```
Comment (nikpachoo) [inline src/components/Layout/index.tsx:32]: Would be great to have the email as a variable.

**Review by nikpachoo — COMMENTED** [review]
"src/images/icon.png" is a default Gasby favicon. Maybe replace it with the Kotlin Logo?

File: src/pages/404.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+const paragraphStyles = {
+  marginBottom: 48,
+}
+const codeStyles = {
+  color: "#8A6534",
+  padding: 4,
+  backgroundColor: "#FFF4DB",
+  fontSize: "1.25rem",
+  borderRadius: 4,
+}
```
Comment (nikpachoo) [inline src/pages/404.js:25]: I would like to offer to put styles in a CSS file.

**Review by nikpachoo — APPROVED** [review]

File: src/pages/404.js (hunk 3 above)
Comment (zoobestik) [reply src/pages/404.js:25]: That's default 404 from gatsby, i think we will do our self. I'll create [task](https://youtrack.jetbrains.com/issue/KTL-593) for tuning...

**Pull request merged**